controllers/horizon_controller.go

```diff
@@ -280,6 +283,35 @@ func (r *HorizonReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return nil
 	}
 
+	// Watch for changes to NADs
+	nadFn := func(ctx context.Context, o client.Object) []reconcile.Request {
+		result := []reconcile.Request{}
+
+		// get all GlanceAPI CRs
+		glanceAPIs := &horizonv1beta1.HorizonList{}
```

Comment:
Is this copy pasta from Glance and the variable name just hasn't been updated?

```suggestion
		horizons := &horizonv1beta1.HorizonList{}
```

Reply:
whoops sorry, bad copy/pasta here :D

Reply:
Updated in the last version.